app/src/main/java/nextstep/signup/ui/component/InputTextField.kt

```diff
@@ -0,0 +1,46 @@
+package nextstep.signup.ui.component
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.text.KeyboardOptions
+import androidx.compose.material3.Text
+import androidx.compose.material3.TextField
+import androidx.compose.material3.TextFieldDefaults
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.text.input.KeyboardType
+import androidx.compose.ui.text.input.PasswordVisualTransformation
+import androidx.compose.ui.text.input.VisualTransformation
+import nextstep.signup.ui.theme.Blue50
+import nextstep.signup.ui.theme.BlueGrey20
+
+@Composable
+fun InputTextField(
+    modifier: Modifier = Modifier,
+    value: String,
+    onValueChange: (String) -> Unit,
+    label: String = "",
+    keyboardOptions: KeyboardOptions = KeyboardOptions.Default
+) {
+    TextField(
+        modifier = modifier
+            .then(
+                Modifier
+                    .fillMaxWidth()
+                    .background(BlueGrey20)),
```

Comment:
```suggestion
        modifier = modifier
            .fillMaxWidth()
            .background(BlueGrey20)),
```
이렇게 작성해도 됩니다! 아래 문서가 Modifier에 대한 이해도를 높이는 데에 도움이 되길 바랄게요 🙂 
https://developer.android.com/develop/ui/compose/modifiers

Reply:
오 그렇게 바로 사용할수도 있군요. 링크 공유감사합니다